packages/core/src/util/nodeJsVersionCheck.js

```diff
@@ -7,7 +7,7 @@
 /**
  * This is the minimum required Node.js version for all @instana packages.
  */
-exports.minimumNodeJsVersion = 14;
+exports.minimumNodeJsVersion = 18;
```

Comment:
qs: Could u pls list all the places where we have defined the min version in the codebase?

IMO we can read this version from package.json here.

Reply:
> IMO we can read this version from package.json here.

But its okay to leave as it is. But I would love to see

> qs: Could u pls list all the places where we have defined the min version in the codebase?

Reply:
packages/core/src/util/nodeJsVersionCheck.js
and all package.json

Reply:
Not in the sh scripts?

Reply:
Refactored the code to read min version from package.json.